server/src/main/java/org/opensearch/extensions/ExtensionsOrchestrator.java

```diff
@@ -60,26 +64,24 @@ public class ExtensionsOrchestrator implements ReportingService<PluginsAndModule
 
     private static final Logger logger = LogManager.getLogger(ExtensionsOrchestrator.class);
     private final Path extensionsPath;
-    final List<DiscoveryExtension> extensionsSet;
+    final Set<DiscoveryExtension> extensionsSet;
+    final Set<DiscoveryNode> extensionsNodeSet;
     TransportService transportService;
-    final DiscoveryNode extensionNode;
+    DiscoveryNode defaultExtensionNode;
```

Comment:
Do we still need this ? 
We do not need a default Extension node, we should always walk through all extensions when invoking extension points.

Reply:
The first extension should not be special :).
With the code we had, it was hacky to enable our manual testing.

We dont need this any more. 
For every extension point we should walk through every extension.